select_test.go

```diff
@@ -222,20 +230,20 @@ func TestMustSql(t *testing.T) {
 	}()
 }
 
-func TestSelectWithoutWhereClause(t *testing.T) {
+func TestSelectWithoutPreWhereOrWhereClause(t *testing.T) {
```

Comment:
Not sure if the rename is needed, these tests also do not have `HAVING`

Reply:
The name seemed to focus on not having a where, hence the rename.